sdk/src/main/java/com/mapbox/maps/CredentialsManager.kt

```diff
@@ -0,0 +1,43 @@
+package com.mapbox.maps
+
+import android.content.Context
+
+/**
+ * Convenience class that holds MapboxMaps related token.
+ * It's possible to create `CredentialsManager` instances as you need them,
+ * however it's convenient to use the default `CredentialsManager.shared`.
+ *
+ * @property accessToken the token will be applied.
+ */
+data class CredentialsManager(val accessToken: String? = null) {
+  /**
+   * Get the access token.
+   * If the supplied token is nil (which is the case for the `shared`),
+   * then we will search for an access token in the application resource.
```

Comment:
```suggestion
   * then we will search for an access token in the application resources.
```
also we could add a link where token is located in resources, I think we have it on Mapbox site.